<!-- Launches delta and two image windows-->

<launch>
	<!-- User arguments -->


	<arg name="video_device"  default="/dev/video1" />
	<arg name="show_image"  default="true" />
	<arg name="name" default="camera"/>

	<!-- camera capture -->






	<!--Fake processor-->
	<node name="camera"
		pkg="ping_pong_robot"
		type="camera"
		output="screen"
		args="1">
		<remap from="/camera/image_in" to="/usb_cam/image_rect_color"/>
		<remap from="/camera/camera_info_in" to="/usb_cam/camera_info"/>
	</node>

	<!-- display in/out image -->
	<group if="$(arg show_image)">
		<!--<node
			name="image_view_raw"
			pkg="image_view"
			type="image_view"
			respawn="false"
			output="screen">
			<remap from="image" to="/usb_cam/image_raw"/>
			<param name="autosize" value="false" />
		</node>-->
		<node
			name="image_view_out"
			pkg="image_view"
			type="image_view"
			respawn="false"
			output="screen">
			<remap from="image" to="/camera/image_out"/>
			<param name="autosize" value="false" />
		</node>
	</group>

</launch>
